### ISSUE TITLE
Catch hover event

### ISSUE THREAD

**Issue by amikrop** [PrismJS/prism#1606]
I was wondering what is the way to catch the hover event on some specific line. I use the line-highlight plugin, so I can see there are some <div aria-hidden="true"> in the element tree, which correspond to the highlighted lines. I try to bind hover events to them but it doesn't work.

**Comment by RunDevelopment** [comment]
`<div aria-hidden="true">` has the property `pointer-events: none`. This is necessary because of the `div` being on top of the text which makes interacting with the text below impossible otherwise.

But one JavaScript-side effect of `pointer-events: none` is that all mouse events are suppressed, so no `mouseover` :(

However, you can use the `mouseover` of the underlying `pre` can calculate if the current pointer position is inside (or "over") the `div`.
